rpc/rpc.go

```diff
@@ -37,5 +41,6 @@ func (c *Context) Copy() *Context {
 		Stopper:      c.Stopper,
 		RemoteClocks: newRemoteClockMonitor(c.localClock),
 		DisableCache: c.DisableCache,
+		HealthWait:   c.HealthWait,
```

Comment:
This method could be changed so we don't have to keep the list of fields up to date:

````
func (c *Context) Copy() *Context {
    copy := *c
    copy.RemoteClocks = newRemoteClockMonitor(c.localClock)
    return &copy
}
```